schemas/+task/Task.m

```diff
@@ -1,16 +1,18 @@
 %{
-task:                varchar(32)
+task:                   varchar(32)
 ---
-task_description='': varchar(512)
+task_description='':    varchar(512)
+table_population='':    BLOB                # list of main tables the task will be populated
```

Comment:
Added a table_population (is a list of "main" tables to be populated with this manipulation)
Same a manipulation_type but for tasks